docs/src/topics/multi-tenancy.md

```diff
@@ -0,0 +1,37 @@
+# Multi-Tenancy
+
+CAPL can manage multi-tenant workload clusters across Linode accounts.
+LinodeCluster objects may reference an optional Secret containing their Linode
+credentials (i.e. API token) to be used for the deployment of "things"
```

Comment:
I would slightly change the wording to note that this is for Linode resources associated with individual workload clusters such as Linodes, VPCs, and NodeBalancers.

Reply:
Better? Worse? 🤔 https://github.com/linode/cluster-api-provider-linode/pull/142/commits/9942590798b51169f5fcd60fb7d223f02e5390c1